Issue #803 fix ‘xAPI specification’ references sections 8.1.2, 8.2.2 and 8.3

MartinKoob opened the pull request:
Issue # This includes  ‘xAPI specification’ references sections 8.1.2, 8.2.2 and 8.3 as per Aug 18, 2023 meeting.

As well changes were made to definitions of xAPI and use of words 'specifications'  vs Standard

MartinKoob committed:
Issue #803 Update cmi5_spec.md with revised definition and references for xAPI spec and standard

As per cmi5 subgroup meeting Aug 18, 2023.

Issue #803 Update cmi5_spec.md with revised definition and references for xAPI spec and standard.

These changes came out of discussion of the references to xAPI in the issue description

MartinKoob committed:
Update spec_cmi5.md issue #803   change to xAPI

Change instances of 'the xAPI Specification' or 'the xAPI' to just 'xAPI'

MartinKoob committed:
Issue #803 Other Instances where changes needed from  ' the xAPI Specification' to 'xAPI' that were not discussed at Aug 18, 2023

Issue #803 Other Instances where changes needed from  ' the xAPI Specification' to 'xAPI' that were not discussed at Aug 18, 2023

MartinKoob commented:
There were additional  instances of  referring to ‘the xAPI specification’ that I found in the document after the Aug 19, 2023 meeting.  I have also changed these instances  to ‘xAPI’.



# 3.0 Definitions 

* __Internationalized Resource Locator (IRL)__: According to the xAPI specification, an IRL is an IRI that, when translated into a URI (according to the IRI to URI rules), is a URL.

* __Learning Records Store (LRS)__: As defined in the xAPI specification.


## 9.1 Statement ID
The AU MUST assign a statement `id` property in UUID format (as defined in the xAPI specification) for all statements it issues.

##9.5.1 Score
The following are properties of the <code>score</code> object per the xAPI specification: 
<ul>

### 10.2.1 contextTemplate

Data Type:	JSON context object as defined in the xAPI Specification.

# 11.0 xAPI Agent Profile Data Model  

In cmi5, Learner Preferences are scoped to the Agent representing the enrolled LMS learner. The agent used in xAPI Agent Profile requests MUST match the actor property generated by the LMS at AU launch time.  Both the LMS and the AU MAY write changes to Learner Preferences in the xAPI Agent Profile.  The LMS/LRS MAY choose to ignore or override Learner Preference changes requested by the AU by returning a "403 Forbidden" response as defined in the xAPI specification.  The AU MUST NOT treat the 403 response as an error condition.

 # 12.0 xAPI Activity Profile Data Model

The AU MAY use the Activity Profile API according to the xAPI specification.

MrBillMcDonald commented:
Per 8/25 Meeting change xAPI reference link to 

change xAPI reference link to https://github.com/adlnet/xAPI-Spec
Change “xAPI specification” references to just “xAPI” (not “xAPI standard”)

MartinKoob committed:
Issue #803 Section 1.1 Scope - reverse changes that referred to 'standard' in relation to xAPI

revert
- 'the xAPI Standard' to -> 'xAPI'

- revert 'this standard' to this spedification'

Rationale:
  - Using 'specification' within the document to refer to the cmi5 specification is okay.
 - Using both 'the xAPI specification' and 'the xAPI Standard' are problematic during the current transition from xAPI 1.0.x to xAPI 2.0 so it was decided to just us 'xAPI'.  The reader can look to the definition of xAPI to see which version that refers to.

MartinKoob committed:
issue #803 revise references to standard back to specification as per Aug 25, 2023 meeting

sorry about the long commit description but I want to make sure I have done everything correctly so I documented it as I went along.

This commit has changes that were decided at the Aug 25, 2023 meeting.

There were discussions at both the Aug 18 and Aug 25, 2023 sub group meetings about how to refer to xAPI in the cmi5 specification I.e. should there be a reference to xAPI 2.0, should it be referred to as a standard or a specification.

At the Aug 18, 2023 meeting it was decided to add the a reference to xAPI 2.0 and update the definition of Experience API (xAPI) to refer to xAPI 2.0.  The new text was added.

References
     * Experience API (xAPI) Standard (IEEE 9274.1.1 xAPI 2.0)
     https://opensource.ieee.org/xapi

Definitions
* __Experience API (xAPI)__: The IEEE 9274.1.1 xAPI 2.0 Standard. A runtime data communication specification for learning content (AU) to send and receive data to a Learning Record Store (LRS). xAPI is used to define the data transport and the data model. As far as requirements found in this document (other than the requirement of xAPI usage), the xAPI Specification 1.0.x is equivalent.

At the Aug 25, 2023 meeting there was further discussion on the topic of  whether to refer to the new xAPI 2.0 standard in the cmi5 Specification and it was decided that it was premature and so the decision was made to reverse the changes made at the Aug 18 meeting.

References
- The new reference to xAPI 2.0 was removed.
- The URL for xAPI in the original reference was changed to
     https://github.com/adlnet/xAPI-Spec

Definitions
The references to xAPI 2.0  and xAPI Specification 1.0.x were removed from the Definition of Experience API (xAPI). Also the second sentence was edited for clarity.  The revised text is as follows:

* __Experience API (xAPI)__: A runtime data communication specification for learning content (AU) to send and receive data to a Learning Record Store (LRS).  xAPI is used to define the data transport and the data model.

Other decisions at the Aug 25, 2023 meeting related to the discussions above.

- Change “xAPI specification” references to just “xAPI” (not “xAPI standard”)s)
- Change “xAPI specification” references (outside of 2.0 References and 3.0 Definitions to just “xAPI”
- Continue to refer to cmi5 as a “specification” on the current Github site.
	- there were a few references to “this standard” referring to the cmi5 specification that I changed to ‘this specification’.

thomasturrell commented on cmi5_spec.md:
```diff
@@ -251,10 +251,10 @@ The following referenced documents are indispensable for the application of this
-* "Experience API", version 1.0.x (subject to change just prior to release), ADL  
-https://github.com/adlnet/xAPI-Spec/blob/xAPI-1.0.2/xAPI.md
+* Experience API (xAPI), version 1.0.3  
+https://github.com/adlnet/xAPI-Spec
```
@andyjohnson Is this the correct location? Should we be linking to https://opensource.ieee.org/xapi/xapi-base-standard-documentation?

thomasturrell replied:
I now understand that this should reference version 1.0.3 not version 2.0

MrBillMcDonald commented:
Approved Per Sept 1, Voting meeting
